cli/main.py

```diff
@@ -81,6 +74,12 @@ def echo_version(ctx, param, value):
 # CLI Configuration
 #
 
+# CLI Constants
+DEBUG = True
+KEYRING_PASSPHRASE_ENVVAR = 'NUCYPHER_KEYRING_PASSPHRASE'
```

Comment:
Hmm, maybe name this a bit different or set it to `os.environ['NUCYPHER_KEYRING_PASSPHRASE']` to make it more clear. I found it a bit confusing until I scrolled down a bit more.

Reply:
Yeah - Do you think its better to read the environment variable here?

Reply:
Potentially, it's fine as is, but I just thought it was a little weird to read like this. You could probably rename the value so it doesn't sound like you're setting a passphrase there.
I'll leave it up to you to decide.

Reply:
At the moment, this environment variable is also used when creating new keyrings.  Is that good & expected behaviour, in your opinion?

Reply:
I think it's _okay_. I just have a hard time trusting Python to do it right (or the user). I would feel more comfortable to see a param passed into the CLI app to tell it to use the env var (or, potentially, _the_ env var) rather than just use it if it is set.